### ISSUE TITLE
Drop Qt 4.x support in 1.4-series

### ISSUE THREAD

**Issue by mkrautz** [mumble-voip/mumble#1605]
We should drop Qt 4.x support and remove our ifdefs when we branch 1.4.x.

**Comment by hacst** [comment]
All for it. As we retain protocol level backwards compatibility in 1.X people on old distros can still use the old version shipped with their distribution if they don't mind having no security updates.

**Comment by davidebeatrici** [comment]
Superseded by #3484.

**Comment by VVD** [comment]
Do mumble support Qt5?

**Comment by davidebeatrici** [comment]
Of course.

**Comment by VVD** [comment]
Thanks.

**Comment by davidebeatrici** [comment]
You're welcome.

**Comment by VVD** [comment]
I read https://wiki.mumble.info/wiki/BuildingLinux#1.2.x and want to ask:
Do 1.2.19 support Qt5 or only since 1.3? Was 1.3 released already?

FreeBSD have port for mumble and murmur:
http://svnweb.freebsd.org/ports/head/audio/mumble/
https://svnweb.freebsd.org/ports/head/audio/murmur/
But Qt4 support in FreeBSD is near EOL - both ports marked with:
DEPRECATED=             Qt4 has been EOL since december 2015
EXPIRATION_DATE=        2019-03-15
Ports will be removed just after this date together with Qt4 and other applications which depends on Qt4. I need release to update it and keep in ports.

Thanks!

**Comment by Kissaki** [comment]
@davidebeatrici 
I would prefer keeping specific task issues open until they are done instead of closing them in favour of generic discussion tickets that wash out the task.

**Comment by davidebeatrici** [comment]
This task is almost done anyway.

As soon as #3644 is merged, I will rebase #3602 and then merge it.
